src/commands/cmd_acl.c

```
@@ -0,0 +1,595 @@
+    } 
+	const char *graph_commands = getenv("GRAPH_USER");
+	if ((graph_commands == NULL) || 
+		(strcasecmp(graph_commands, "false") == 0)
+		|| (strcmp(graph_readonly_commands, "") == 0)) {
+		return REDISMODULE_ERR;
+    
+    } 
+	const char *graph_admin_commands = getenv("GRAPH_ADMIN");
+	if ((graph_admin_commands == NULL) || 
+		(strcasecmp(graph_admin_commands, "false") == 0)
+		|| (strcmp(graph_readonly_commands, "") == 0)) {
```

Cppcheck (2.10-2)

[error] Memory leak

(memleak)